packaging/os/apt.py

```diff
@@ -546,7 +548,7 @@ def main():
             package = dict(default=None, aliases=['pkg', 'name'], type='list'),
             deb = dict(default=None),
             default_release = dict(default=None, aliases=['default-release']),
-            install_recommends = dict(default='yes', aliases=['install-recommends'], type='bool'),
+            install_recommends = dict(default='default', aliases=['install-recommends'], choices=['default', 'yes', 'no']),
```

Comment:
make the option type='bool' and the default to None, otherwise plays that already use true/false or Yes/No will start failing.